icons/home-plug.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jguddas",
+    "karsa-mistmere"
```

Comment:
```suggestion
    "jguddas"
```

I think there were definitely some other contributors there (I mostly just optimized stuff in #2116), although I'm finding it hard to decide whether they should even be kept, since you've basically redesigned the entire icon from scratch. 🤔

Reply:
I did not base the redesign on the current home icon.

Reply:
Yeah, that's also my point. :)